### PR TITLE
Rename to CodeComplianceTest_oM

### LINKED ISSUE #291: Rename Test_oM to CodeCompliance_oM
Following discussion [here](https://github.com/BHoM/BHoM/discussions/1061)

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
### NOTE: Depends on 
https://github.com/BHoM/BHoM/pull/1077

 ### Issues addressed by this PR
Fixes #291

### CODE REVIEW

**Review by IsakNaslundBh — CHANGES_REQUESTED** [review]
Some comments below.

Also, all the files are still in the Test_oM folder. That needs to be updated.

File: Test_oM/Interoperability/Enums/ComparerType.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 using System.ComponentModel;
 
-namespace BH.oM.Test.Interoperability
+namespace BH.oM.CodeComplianceTest.Interoperability
```
[the remark this answers is not in the document]
Comment (FraserGreenroyd) [reply Test_oM/Interoperability/Enums/ComparerType.cs:25]: Can I do that one in another issue, I'd like to get this merged so I can update a nuget for BHoMBot - the nuget does not know of InteroperabilityTest_Engine so will not be impacted?

**Comment by FraserGreenroyd** [comment]
@BHoMBot check project compliance

File: Test_oM/Interoperability/Enums/ComparerType.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (IsakNaslundBh) [reply Test_oM/Interoperability/Enums/ComparerType.cs:25]: no, but all my scripts are broken and can not be used without being remade, and others are starting to use them as well. Better get this sorted now?

[the remark this answers is not in the document]
Comment (FraserGreenroyd) [reply Test_oM/Interoperability/Enums/ComparerType.cs]: Fineeeeeeeeeee

image


:wink:

File: CodeComplianceTest_Engine/Query/Checks/HasValidCopyright.cs [hunk 2]
```diff
@@ -20,8 +20,8 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using BH.oM.Test;
-using BH.oM.Test.Attributes;
+using BH.oM.CodeComplianceTest;
```
[the remark this answers is not in the document]
Comment (FraserGreenroyd) [reply CodeComplianceTest_Engine/Query/Checks/HasValidCopyright.cs:23]: Done

**Review by IsakNaslundBh — DISMISSED** [review]
Happy now

**Review by michaelhoehn — APPROVED** [review]
LGTM

**Pull request merged**